components/hero/hero.module.scss

```diff
@@ -0,0 +1,99 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+.container {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 100px auto;
+  max-width: $max-content;
+  overflow: hidden;
+  position: relative;
+
+  @include breakpoint(medium-only) {
+    padding: 0 1rem;
+  }
+
+  @include breakpoint(small) {
+    align-items: flex-start;
+    flex-direction: column;
+    margin: 0;
+    padding: 100px 1rem;
+    text-align: center;
+  }
+}
+
+.row {
+  display: flex;
+}
+
+.info {
+  width: 60%;
+  z-index: 2;
+
+  @include breakpoint(small) {
+    width: 100%;
+  }
+}
+
+.title {
+  @include h1;
+}
+
+.subtitle {
+  @include h3;
+
+  margin: 20px 0 50px;
+}
+
+.buttons {
+  display: flex;
+
+  @include breakpoint(small) {
+    flex-direction: column;
+  }
+}
+
+.getStarted {
```

Comment:
Please rename this to `get-started` according to BEM convention

Reply:
I think we can avoid that here. The CSS modules convention is to use [camel case](https://github.com/css-modules/css-modules#naming), as used above. And also, since all classes are locally scoped to the file, BEM isn't too necessary in our case.

Reply:
Thanks for sharing the github page - while I understand that css-modules might have recommended camel case, it is not enforced. 

I hope you don't mind my honesty, but I do find it quite odd first of all as this new naming convention isn't particularly brought up, and it has been our team norm to use BEM convention for css naming ( with many good reasons to, given that it is a [recommended best practice](https://cssguidelin.es/#naming-conventions) to follow for style naming). 

In looking at how the variables are set up in `variables.scss` here, it seems like we are adopting BEM naming here, so I am quite confused. 

While we might be using css-modules as a means to import styles in next.js, ultimately we are using scss which generally advocates [hyphen-delimited naming for classes ](https://sass-guidelin.es/#naming-conventions) - it doesn't to make sense to deviate from it just because we are using css-modules as a mechanism for style import.

Reply:
I understand. I've changed it to a single-named class now.

To just touch on some of your other points: for the team norms point of view, I think we should be empowered and excited to try new things in this project. We don't need to carry over every convention from Kedro-Viz.

As for naming, I'm only following the convention that [Next.js documents](https://nextjs.org/docs/basic-features/built-in-css-support#sass-variables). They detail a SCSS variable, written with a hyphen, and then exported and used in JSX with camelCase. The same pattern would be used if it was a class name.